api_client/python/timesketch_api_client/sketch.py

```diff
@@ -1085,3 +1097,47 @@ def add_event(self, message, timestamp, timestamp_desc):
             self.api.api_root, self.id)
         response = self.api.session.post(resource_url, json=form_data)
         return response.json()
+
+    def is_archived(self):
+        """Return a boolean indicating whether the sketch has been archived."""
+        if self._archived is not None:
```

Comment:
Why not simply:
```
if not self._archived:
  return
```

Reply:
because I want to specifically check whether the value is None... otherwise I'm checking a boolean.

if this is None,; then I know I haven't actually checked, and then it is time for me to check using the API, if it is True/False, then I know I've checked before and I can just return that value (that is this is just checking the initial value)

Reply:
ok